src/platform/forms-system/src/js/definitions/content/directDeposit.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+
+export const directDepositDescription = () => {
+  return (
+    <div>
+      <p>
+        We make payments only through direct deposit, also called electronic
+        funds transfer (EFT). If you’re approved for the Rogers STEM
+        Scholarship, we’ll need to know where to deposit the funds.
```

Comment:
Can we remove the sentence "If you're approved for the Rogers STEM Scholarship, we'll need to know where to deposit the funds." @cvalarida

Reply:
:joy: Yeah, that...yeah. :see_no_evil: :sweat_smile: